File: Java/poo/salary/library/arl.java
package poo.salary.library;

public class arl {

    private double valueArl;

    public arl(double valueArl) {
        this.valueArl = valueArl;
    }

    public double getvalueArl() {
        return this.valueArl;
    }

    public double calculateDiscount() {
        return (this.getvalueArl() * 0.005);
    }
}

File: Php/poo/salary/library/arl.php
<?php

    class arl extends person {
    private $valueArl;
    
    public function __construct($valueArl) {
        $this->valueArl = $valueArl;
    }

    public function getValueArl(): float  {
        return $this->valueArl;
    }

    public function calculateDiscount(): float  {
        $this->calculate = ($this->getValueArl() * 0.005);
        return $this->calculate;
    }
}